src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -100,15 +90,11 @@ public void InitializeService(ServiceHost serviceHost)
             {
                 DatabaseTaskHelper helper = AdminService.CreateDatabaseTaskHelper(connInfo, databaseExists: true);
                 SqlConnection sqlConn = GetSqlConnection(connInfo);
-                if (sqlConn != null)
+                if ((sqlConn != null) && !connInfo.IsSqlDW)
```

Comment:
so we should add the check here too maybe

---
In reply to: [124131526](https://github.com/Microsoft/sqltoolsservice/pull/396#discussion_r124131526) [](ancestors = 124131526)